prow/cmd/deck/main.go

```diff
@@ -1022,7 +1023,14 @@ lensesLoop:
 	prHistLink := ""
 	org, repo, number, err := sg.RunToPR(src)
 	if err == nil && !cfg().Deck.Spyglass.HidePRHistLink {
-		prHistLink = "/pr-history?org=" + org + "&repo=" + repo + "&pr=" + strconv.Itoa(number)
+		PRHistLinkTemplate := cfg().Deck.Spyglass.PRHistLinkTemplate
```

Comment:
local variables should be lowerCamelCase, right? (Just like prHistLink)

Reply:
good point, updated